src/Chapter1.hs

```diff
@@ -632,8 +642,12 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n = penultimateNum + lastNum
+  where
+    lastNum = absN `mod` 10
+    penultimateNum = (absN `div` 10) `mod` 10
```

Comment:
👍🏼 

In here you use both:

```haskell
absN `mod` 10
absN `div` 10
```

One hint to make your solution even shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that is actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(a, b) = absN `divMod` 10
```

You can see how we could pattern match on the pair 🙂